src/Diagnostics.HealthChecks.AspNetCore/Readme.md

```diff
@@ -0,0 +1,11 @@
+## Health Check ASP.NET Core helpers and extensions
+
+This project introduces features specifically dependent from the ASP.NET Core WebAPI Framework. This will not introduce other composable classes, but it will leverage them to provide a new factory method to create a **Liveness Health Check**, an health check that will only verify that the endpoint is reachable without letting it execute its logic, short-circuiting it using an implementatin of the `ActionFilterAttribute`.
```

Comment:
I can't interpret the intent from the first line. Does this make sense?
```suggestion
This project introduces features specifically designed for the ASP.NET Core WebAPI Framework. This will not introduce other composable classes, but it will leverage them to provide a new factory method to create a **Liveness Health Check**, an health check that will only verify that the endpoint is reachable without letting it execute its logic, short-circuiting it using an implementatin of the `ActionFilterAttribute`.
```

Reply:
It does, it expresses the sense more clearly than what I wrote